configure.ac

```diff
@@ -90,6 +90,20 @@ fi
 
 PKG_CHECK_MODULES(COPY_WORKER, gio-2.0 gnome-keyring-1)
 
+# Zint barcode
```

Comment:
I think this should be squashed with the commit that introduces the factory dialog.